assets/tasks/common.json

```diff
@@ -641,6 +640,26 @@
                 "color": "terminal.ansiYellow"
             }
         },
+        {
+            "label": "remove-dangling-images",
```

Comment:
did you saw some impact in the performance? Because, ok that we leave some dangling behind, but for the same hashes this should improve the performance on the next push

Reply:
As far as I observed this task tends to run quickly, because there is not much to remove when something is changed on the source code which is pretty much the last layers that are small.
On the automated tests I saw a very nice improve of performance actually. As in those tests all the projects created have the same name, removing the older one improved the performance, and also prevented the problem of [no space left on the device]

Reply:
But if you could test it on your side also would be nice to also make sure this task doesn't bring any side effects.

Reply:
If there are many dangling images at the first time it can take some seconds, but for the second time and beyond it is fast